docker-compose-3-persona.yml

```diff
@@ -83,16 +83,19 @@ services:
       - hydrogen-producer-node
       - hydrogen-producer-postgres-identity
     environment:
-      - PORT=9000
-      - API_HOST=hydrogen-producer-node
-      - API_PORT=9944
-      - DB_HOST=hydrogen-producer-postgres-identity
-      - DB_PORT=5432
-      - DB_NAME=dscp-identity
-      - DB_USERNAME=postgres
-      - DB_PASSWORD=postgres
-      - SELF_ADDRESS=5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY
-      - AUTH_TYPE=${AUTH_TYPE:-NONE}
+      - 'PORT=9000'
```

Comment:
Why do these all need wrapping with ' They worked fine before?

Only add to the new API_SWAGGER stuff if necessary for escaping spaces

Reply:
On it ..